### PR TITLE
Ajit_Assignment_26_Done_25mins

### PR DESCRIPTION
**Pull request by AjitSJune20**
image

### CODE REVIEW

File: src/ajitSJune20/SumOfNumInString.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		int sum = 0;
+		String temp = "";
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			if (Character.isDigit(ch))
+				temp += ch;
+			else {
+				if (temp.length() != 0) {
+					sum += Integer.parseInt(temp);
+					temp = "0";
```
Comment (KartikeyDJune20) [inline src/ajitSJune20/SumOfNumInString.java:15]: If you are reinializing temp with "0".. that means your length will never gonna be zero again, so there is no point for the if statement above to check the length

Comment (KartikeyDJune20) [reply src/ajitSJune20/SumOfNumInString.java:15]: Oh got it for the first time it will need that..

Comment (AjitSJune20) [reply src/ajitSJune20/SumOfNumInString.java:15]: Thanks.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**